### ISSUE TITLE
FWIW: The Intercept's Article & Docs for a Stingray

### ISSUE THREAD

**Issue by TPS** [CellularPrivacy/Android-IMSI-Catcher-Detector#898]
[The Intercept has published an article on September 12, 2016](https://theintercept.com/2016/09/12/long-secret-stingray-manuals-detail-how-police-can-spy-on-phones/) & includes web viewers to docs with some info on a Stingray's details.

Here are the links & files included:

https://assets.documentcloud.org/documents/3105793/Gemini-3-3-Quick-Start-Guide.pdf
https://assets.documentcloud.org/documents/3105849/Gemini-RayFish-Controller-R3-3-1-Release-Notes.pdf
https://assets.documentcloud.org/documents/3105641/iDEN-2-4-Operator-Manual.pdf
https://assets.documentcloud.org/documents/3105806/iDEN-2-4-Release-Notes.pdf
https://assets.documentcloud.org/documents/3105805/Arrowhead-1-0-1-Release-Notes.pdf

[Arrowhead-1-0-1-Release-Notes.pdf](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/files/476174/Arrowhead-1-0-1-Release-Notes.pdf)
[Gemini-3-3-Quick-Start-Guide.pdf](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/files/476175/Gemini-3-3-Quick-Start-Guide.pdf)
[Gemini-RayFish-Controller-R3-3-1-Release-Notes.pdf](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/files/476176/Gemini-RayFish-Controller-R3-3-1-Release-Notes.pdf)
[iDEN-2-4-Operator-Manual.pdf](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/files/476177/iDEN-2-4-Operator-Manual.pdf)
[iDEN-2-4-Release-Notes.pdf](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/files/476178/iDEN-2-4-Release-Notes.pdf)

https://drive.google.com/folderview?id=0B9U-jNlL-z-uRldMYUV0Q0ZRbFU

**Comment by webcoderz** [comment]
how far have u gotten with reverse engineering?

**Comment by TPS** [comment]
Not at all, _I_'m not an Android coder, but I figured posting this would help _them_.

**Comment by webcoderz** [comment]
Yea I'm not either I am a novice python programmer and going to school for Computer science . I also like embedded macines so it sounds like it could be a neat project

**Comment by He3556** [comment]
thanks @TPS for the links. I read it all and this manual describes a basic IMSI catcher (identifying and tracking) without the possibility to intercept voice and data transmitions. The new generation has other features that i am missing in this manual. But still a great confirmation thx

**Comment by mimi89999** [comment]
I think that the @TPS wanted to link to that article: https://theintercept.com/2016/09/12/long-secret-stingray-manuals-detail-how-police-can-spy-on-phones/
(the link is broken.)

**Comment by TPS** [comment]
@mimi89999 Huh, both still WFM identically. 🤷

**Comment by mimi89999** [comment]
The very first one: "The Intercept has..." points to nowhere.

**Comment by TPS** [comment]
WTH?! I check on my mobile, all is ok: source link & all.  On my laptop, it's as you say: no source link. :confounded:  Anyway, edited the link back in.  All ok for you now?

**Comment by mimi89999** [comment]
Works now. Thanks.
